index.js

```diff
@@ -10,4 +10,8 @@ https://github.com/fluid-project/first-discovery-server/raw/master/LICENSE.txt
 
 "use strict";
 
+var fluid = fluid || require("infusion");
+var gpii = fluid.registerNamespace("gpii");
 require("./src/js/firstDiscoveryServer.js");
+
+gpii.firstDiscovery.server();
```

Comment:
Now you've moved the startup into this driver file, I think this is a good place for making the configuration for this server to be sensitive to the vagrant configuration for the port - which is the ultimate authoritative upstream value. @avtar  has arranged for it to be read into an environment variable. A reasonable approach would be to send a distributeOptions block into the arguments of this function call which forwards the port number to the correct server. You should also consult with @duhrer about whether configuration for gpii-express can be improved so that we don't need to write the port number in two places. What happens if we leave out the "baseUrl"?